pypeit/spectrographs/lbt_mods.py

```diff
@@ -377,6 +456,35 @@ def __init__(self):
         self.camera = 'MODS2R'
         self.numhead = 1
 
+    def get_detector_par(self, hdu, det):
```

Comment:
If these and other instruments are now to be supported, add them to the list of spectrographs and the  `README` at the top.

Reply:
Not sure where you are referring to here?

Reply:
https://github.com/pypeit/PypeIt/blob/master/README.md
https://github.com/pypeit/PypeIt/blob/master/doc/spectrographs.rst

Reply:
Added.